src/Applications/Applications/examples/v1.0/Get-MgUserAppRoleAssignment.md

```diff
@@ -1,18 +1,24 @@
-### Example 1: {{ Add title here }}
-```powershell
-PS C:\> {{ Add code here }}
-
-{{ Add output here }}
-```
+### Example 1: Get assigned user roles
 
-{{ Add description here }}
+This command gets all the application roles that the selected user has been assigned.
 
-### Example 2: {{ Add title here }}
 ```powershell
-PS C:\> {{ Add code here }}
-
-{{ Add output here }}
+ Get-MgUserAppRoleAssignment -UserId "529827aa-d058-4821-a012-4de3ce093955" | 
+  Format-List Id, AppRoleID, CreationTimeStamp, PrincipalDisplayName,PrincipalId, PrincipalType, ResourceDisplayName
 ```
 
-{{ Add description here }}
+```Output
```

Comment:
We should remove ```Output as its not part of the structure, it will also cause a split in the code block like illustrated below.
![image](https://user-images.githubusercontent.com/1641829/121788509-7584a800-cbd6-11eb-92da-3787f3bac0d1.png)

As such the format should be as indicated below. 
### Example 1: {{ Add title here }}
```powershell
PS C:\> {{ Add code here }}

{{ Add output here }}
```

{{ Add description here }}